Projects/Features/Scene/GroupDetailScene/Vote/VoteComplete/VoteCompleteView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  VoteCompleteView.swift
+//  GroupDetail
+//
+//  Created by 최혜린 on 7/6/24.
+//  Copyright © 2024 com.mashup.gabbangzip. All rights reserved.
+//
+
+import ComposableArchitecture
+import DesignSystem
+import SwiftUI
+
+struct VoteCompleteView: View {
+  private let store: StoreOf<VoteCompleteCore>
+
+  init(store: StoreOf<VoteCompleteCore>) {
+    self.store = store
+  }
+
+  var body: some View {
+    VStack {
```

Comment:
P2:
```suggestion
    VStack(spacing: 0) {
```
Spacing을 0으로 설정하지 않은 것은 의도된 것일까요?

Reply:
@zekexros 앗 아뇨 깜빡한 것 같습니다! 반영할게요!